### ISSUE TITLE
Runtime exception 'Cannot deserialize XBF metadata property' thrown after upgrading to Template 10 1.1

### ISSUE THREAD

**Issue by RichDef77** [Windows-XAML/Template10#558]
Hello,

After upgrading to Template 10 ver 1.1 I encountered a runtime error:

Exception thrown: 'System.IO.FileLoadException' in Template10Library.dll
Exception thrown: 'Windows.UI.Xaml.Markup.XamlParseException' in WorxHome.UI.exe
WinRT information: Cannot deserialize XBF metadata property list as 'Type' was not found in type 'WorxHome.UI.NavType'. [Line: 0 Position: 0]
An exception of type 'Windows.UI.Xaml.Markup.XamlParseException' occurred in WorxHome.UI.exe but was not handled in user code
WinRT information: Cannot deserialize XBF metadata property list as 'Type' was not found in type 'WorxHome.UI.NavType'. [Line: 0 Position: 0]
Additional information: The text associated with this error code could not be found.

Cannot deserialize XBF metadata property list as 'Type' was not found in type 'WorxHome.UI.NavType'. [Line: 0 Position: 0]

Please note: The app works perfectly fine using the 1.0 version of Template 10

This NavType class is a very simple custom class that appears in our AppShell.xaml page that contains the SplitView control. It is used in conjunction with SplitView radio buttons to pass Navigation data to the NavigationService.

 public class NavType
    {
        public Type Type { get; set; }
        public string Parameter { get; set; }
    }

Thank You!

**Comment by realworld666** [comment]
Similar I'm getting this in 1.1.1. Works fine in 1.0.8

An exception of type 'Windows.UI.Xaml.Markup.XamlParseException' occurred in CycleStreetsUniversal.exe but was not handled in user code

WinRT information: Cannot deserialize XBF metadata type list as 'HamburgerMenu' was not found in namespace 'Template10.Controls'. [Line: 0 Position: 0]

Additional information: The text associated with this error code could not be found.

**Comment by RichDef77** [comment]
I think this may be an issue related to how the NuGet package was created.

**Comment by angusbreno** [comment]
See https://github.com/Windows-XAML/Template10/issues/547... My problem was with Behaviors...

**Comment by gregstoll** [comment]
Adding a reference to the Microsoft.Xaml.Behaviors.Uwp.Managed NuGet fixed this for me as well.

**Comment by RichDef77** [comment]
After manually adding the following Nuget packages I was able to get everything up and running:

1) Manually add Microsoft.Xaml.Behaviors.Uwp.Managed NuGet package v1.03
2) Manually add Newtonsoft.Json NuGet package v8.0.2

Note: You must add these via NuGet. 

Kudos to Jerry Nixon and this community! I opened this issue late Friday afternoon thinking that I'm not going to upgrade to latest version of Template 10 in this release of my project. I come in Monday and the problem is solved!!! Awesome!
